### ISSUE TITLE
AWS: handle case better when volume is 'creating'

### ISSUE THREAD

**Issue by justinsb** [kubernetes/kubernetes#15058]
When we try to attach an EBS volume that is still creating, we get this error around line 1145:

`IncorrectState: vol-79ebdc89 is not 'available'.`

State == `creating` when this happens.

We should probably fix the log messages for this case.

**Comment by a-robinson** [comment]
Is this related to [today's Stack Overflow question](http://stackoverflow.com/questions/32894930/kubernetes-aws-ebs-timeout-waiting-for-volume-state)?

**Comment by justinsb** [comment]
Actually @a-robinson I think that is #14493.  I have a branch (https://github.com/justinsb/kubernetes/tree/aws_integration) where I'm cleaning up a lot of these problems. I think also that I'd like to try again to not copy-and-paste the GCE volume code to AWS, but instead reuse it for AWS (the two have diverged a long way...)  Hopefully we can get #14493 merged and then I have a few more edge-cases I've found.  None as bad as the mistake I made that #14493 is fixing, but important to fix anyway....
